igniter/tools.py

```diff
@@ -14,6 +15,305 @@
     ConfigurationError,
     OperationFailure
 )
+from cloudpathlib import AnyPath
```

Comment:
Code specific for dropbox should be in separated file and only code that is used here should be imported. There is a lot of code which is dropbox specific but does not make sense in context of `igniter/tools` for example class `FilePath` is dropbox specific but I this context it is confusing.